forge-game/src/main/java/forge/game/phase/Untap.java

```diff
@@ -168,10 +168,10 @@ private void doUntap() {
 
         cardsWithKW.addAll(cardsWithKW2);
         for (final Card cardWithKW : cardsWithKW) {
-            if (cardWithKW.isExertedBy(player)) {
+            if (!cardWithKW.canUntap(player)) {
```

Comment:
this line is redundant now because untap call below would just return false anyway